go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/lib/pq v1.2.0
 	github.com/m3db/prometheus_client_golang v0.8.1
 	github.com/olekukonko/tablewriter v0.0.4
-	github.com/olivere/elastic v6.2.21+incompatible
+	github.com/olivere/elastic v6.2.37+incompatible
```

Comment:
In general it'd be better if we do these version upgrades in a standalone diff, as it makes it clearer and easier to see. Probably ok in this diff however.

Reply:
It is good to know! I will follow the way next time.